src/DiffEqWebBase.jl

```diff
@@ -17,7 +17,7 @@ module DiffEqWebBase
     QuickODEProblem{typeof(u0),eltype(tspan),true,typeof(callback)}(f,u0,tspan,callback)
```

Comment:
the fields of these types aren't correct anymore.

Reply:
Yeah, I realized it by the error. Can you help me with the correct one?

Reply:
https://github.com/JuliaDiffEq/DiffEqBase.jl/blob/master/src/problems/ode_problems.jl#L16-L29